### PR TITLE
feat: add effective ACL

### LINKED ISSUE #1013: ACL tab must show effective ACL
Left ACL tab must display inhereted ACL (from parents) and ACL set for the selected schema object

### PR DESCRIPTION
**Pull request by Raubzeug**
Closes #1013 
[Stand](https://nda.ya.ru/t/3yMpAUbi773jS4)

### CODE REVIEW

File: src/containers/Tenant/Acl/Acl.scss [hunk 1]
```diff
@@ -2,21 +2,16 @@
...
-        padding-bottom: 25px;
-    }
     &__result {
-        padding-bottom: 16px;
-    }
-
-    &__owner {
-        font-weight: 600;
+        padding-bottom: var(--g-spacing-4);
+        padding-left: var(--g-spacing-2);
```
Comment (artemmufazalov) [inline src/containers/Tenant/Acl/Acl.scss:7]: Why do we need this padding?

Screenshot 2024-07-22 at 12 34 25

File: src/containers/App/App.scss [hunk 2]
```diff
@@ -117,3 +117,7 @@ body,
 .g-popup {
     max-width: 500px;
 }
+
+.gc-definition-list__title {
+    @include subheader-2-typography();
+}
```
Comment (artemmufazalov) [inline src/containers/App/App.scss:123]: Why do you override this styles? You don't use group titles from `DefinitionList`, you have `<div className={b('list-title')}>{i18n('title_effective-rights')}</div>`

File: src/containers/Tenant/Acl/Acl.scss [hunk 3]
```diff
@@ -2,21 +2,16 @@
...
     }
     &__definition-content {
         display: flex;
         flex-direction: column;
     }
-    &__group-label {
-        @include subheader-2-typography();
+    &__list-title {
+        margin: var(--g-spacing-3) 0 var(--g-spacing-5);
+        @include subheader-3-typography();
```
Comment (artemmufazalov) [inline src/containers/Tenant/Acl/Acl.scss:15]: It differs from titles we have in `Overview` tab. There we have `body-2-typography` and `font-size: 600`. It's defined in two places:

https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/styles/mixins.scss#L95
https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/components/InfoViewer/InfoViewer.scss#L2

I think, all these titles should be similar

Screenshot 2024-07-22 at 12 40 19

Screenshot 2024-07-22 at 12 40 14

**Review by artemmufazalov — CHANGES_REQUESTED** [review]

File: src/containers/Tenant/Acl/Acl.scss (hunk 1 above)
Comment (Raubzeug) [reply src/containers/Tenant/Acl/Acl.scss:7]: Discussed with @adameat that it looks prettier with small padding.

File: src/containers/App/App.scss (hunk 2 above)
Comment (Raubzeug) [reply src/containers/App/App.scss:123]: For now yes, but it is possible to see groups from Definition list (if some props are not default).
Nonetheless, some fixes for styles are needed!

**Review by artemmufazalov — APPROVED** [review]

**Pull request merged**